### PR TITLE
Screencasting advanceduse language reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: userguide/advanceduse/screencasting.rst [hunk 1]
```diff
@@ -1,51 +1,52 @@
...
 
 
-  Prepare your computer to listen to a tcp port(here 1234) and forward raw stream to a video player (here mplayer) with a frame size of 384x640::
+  Prepare your computer to listen to a TCP port (1234 here) and forward the raw stream to a video player (MPlayer here) with a framesize of 384x640::
 
     nc -l -p 1234 | gzip -dc | mplayer -demuxer rawvideo -rawvideo w=384:h=640:format=rgba -
 
 For stream recording:
 
-  Prepare your computer to listen to a tcp port(here 1234), ungzip and forward raw stream to a video encoder (here mencoder)::
```
Comment (comradekingu) [inline userguide/advanceduse/screencasting.rst:33]: Meant to use UNGZIP in the example?

File: userguide/advanceduse/screencasting.rst [hunk 2]
```diff
@@ -1,51 +1,52 @@
...
+You can catch output directly from the ``adb exec-out`` command and forward it to MPlayer::
 
   adb exec-out timeout 120 mirscreencast -m /run/mir_socket --stdout --cap-interval 2 -s 384 640 | mplayer -demuxer rawvideo -rawvideo w=384:h=640:format=rgba -
   
-NB: ``timeout`` here is used in order to kill process properly on device ( here 120 seconds ). Otherwise process still continuing even if killed on computer.
-You can reduce or increase frame per second with``--cap-interval`` (1 = 60fps, 2=30fps, ...)  and size of frames ``384 640`` means width=384 height=640
+``timeout`` above is used to kill the process in a proper manner on the UBports device (120 seconds here).
+(Otherwise the process continues even if killed on the computer.)
+Reduce or increase the number of frames per second with``--cap-interval`` (1 = 60fps, 2=30fps, …)
+and the size of frames ``384 640`` means a width of 384 px and a height of 640 px.
```
Comment (comradekingu) [inline userguide/advanceduse/screencasting.rst:17]: Is this N64 resolution?

**Review by doniks — APPROVED** [review]

**Comment by doniks** [comment]
Grrr. Now sphinx/git have some problem ... Not your fault I think. Not sure how to solve this right now :(

**Comment by doniks** [comment]
Mhm, @comradekingu I see that the master branch on your fork of the it docs is really old. Now, I think this *should not* matter, but maybe this gets sphinx/gut confused. Can you update that branch?

**Comment by comradekingu** [comment]
@doniks Good now?

**Pull request merged**